### PR TITLE
[Add][13.0][MIG] account_banking_ach_credit_transfer

### PR DESCRIPTION
**Pull request by Chanakya-SerpentCS**

### CODE REVIEW

**Comment by bodedra** [comment]
@Chanakya-SerpentCS Can you make CI happy? Also preserve commit history. Ref: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by Chanakya-SerpentCS** [comment]
@bodedra CI will be green once #65 is merged.

**Review by bodedra — APPROVED** [review]
Functional reviewed locally. :+1:

**Pull request closed without merging**
